cep-recipe-jinja.md

```diff
@@ -0,0 +1,125 @@
+# Jinja functions in recipes
+
+The new recipe format has some Jinja functionalities. We want to specify what functions exist and their expected behavior.
+
+## The compiler function
+
+The compiler function is used to stick together a compiler from {lang}_compiler and {lang}_compiler_version
+
+The function looks as follows:
+
+```yaml
+${{ compiler('c') }}
+```
+
+This would pull in the c_compiler and c_compiler_version from the variant config. The compiler function also adds the target_platform to render to something such as:
```

Comment:
```suggestion
This would pull in the `c_compiler` and `c_compiler_version` values from the variant config. The compiler function also adds the `target_platform` value to render to something such as:
```